src/style.css

```
@@ -0,0 +1,24 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 0;
```

it will be clearer to use `center`
```suggestion
  background-position: center;
```